scripts/balancing.lua

```diff
@@ -1980,17 +1980,17 @@ DefineAnimations(
 )
 
 for i,spec in ipairs({
-   { Var = "Supply", Gold = 700, Unit = "brigand" },
-   { Var = "Demand", Gold = 1200, Unit = "ogre" },
+   { Var = "Supply", Gold = 1000, Wood = 4000, Unit = "brigand" }, --wood is used for cooldown. So long to reduce micro.
+   { Var = "Demand", Gold = 1200, Wood = 6000, Unit = "ogre" }, -- 1000 cooldown time = 200 production time (footman)
 }) do
    DefineSpell("spell-hire-" .. spec.Unit,
-      "manacost", 0, "range", 0, "target", "self", "cooldown", 5, "action", {
+      "manacost", 0, "range", 0, "target", "self", "cooldown", spec.Wood, "action", {
          { "lua-callback", function(ident, caster, goalX, goalY, target)
-            local value = GetUnitVariable(caster, spec.Var, "Max")
-            if value < 40 then
-               AddMessage("No one here looking for a job right now...")
-               return false
-            end
+            --local value = GetUnitVariable(caster, spec.Var, "Max")
```

Comment:
I would suggest you add this back and just use a different threshold value if you think the value is too low or high

Reply:
yes, 

but cooldown have nice animation on button, and player see how much tile left to another hire